api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -755,4 +755,38 @@ public void copyFile_shouldCopyInputstreamToOutputstreamAndCloseTheOutputstream(
 		verify(output, times(1)).close();
 	}
 
+	/**
+	 * @see OpenmrsUtil#isTestMode()
+	 */
+	@Test
+	public void isTestMode_shouldPassIfFunctionalTestModeIsEnabled() {
+		
+		System.setProperty("FUNCTIONAL_TEST_MODE", Boolean.toString(true));
+		boolean isTestMode = OpenmrsUtil.isTestMode();
+
+		assertTrue(isTestMode);
+	}
+
+	/**
+	 * @see OpenmrsUtil#isTestMode()
+	 */
+	@Test
+	public void isTestMode_shouldPassIfFunctionalTestModeIsDisabled() {
+
+		System.setProperty("FUNCTIONAL_TEST_MODE", Boolean.toString(false));
+		boolean isTestMode = OpenmrsUtil.isTestMode();
+
+		assertFalse(isTestMode);
+	}
+
+	/**
+	 * @see OpenmrsUtil#getRuntimePropertiesFilePathName(applicationName)
+	 */
+	@Test
+	public void getRuntimePropertiesFilePathName_shouldPassIfPropertyFileNotExist() {
+		
+		String pathName = OpenmrsUtil.getRuntimePropertiesFilePathName("app-openmrs");
+		assertNull(pathName);
+	}
+
```

Comment:
Nice Work @IsurangaPerera . I suggest you to initiate a [code review](https://wiki.openmrs.org/display/docs/Code+Review) :)

Reply:
@PKatGITHUB Yeah I'm working on extending the coverage. Sure will initiate a code review. thanks